### PR TITLE
issue #266 fix

### PR DESCRIPTION
**Pull request by ashish0910**
Preview is added while taking a picture in Record activity .

### CODE REVIEW

**Comment by llaske** [comment]
Don't work on my side: no preview displayed in Chrome 72 and do an error on console in Firefox 65.
What's your testing environment?

**Comment by ashish0910** [comment]
@llaske added support for Firefox 65 and Chrome 72 .

**Comment by llaske** [comment]
It's now work on Chrome but not on Firefox: the photo is displayed but it never recorded. There is an error on console:

```
  Recorder state changed: recording recordrtc.js:705:17
  Initialized recorderType: MediaStreamRecorder for output-type: video recordrtc.js:99:13
  Stopped recording video stream. recordrtc.js:125:13
  Recorder state changed: stopped recordrtc.js:705:17
video/webm -> 217 Bytes recordrtc.js:166:17
  La ressource média data:video/webm;base64,GkXfowEAAAAAAAAfQoaBAUL3gQFC8oEEQvOBCEKChHdlYm1Ch4ECQoWBAhhTgGcB/////////xFNm3QBAAAAAAAAABVJqWYBAAAAAAAASSrXsYMPQkBEiYgAAAAAAAAAAE2AmFFUbXV4aW5nQXBwTGliV2ViTS0wLjAuMVdBmVFUd3JpdGluZ0FwcExpYldlYk0tMC4wLjEWVK5rAQAAAAAAADWuAQAAAAAAACzXgQFzxYQ+kgeYJYaIg1ZQOIOBAYaFVl9WUDjgAQAAAAAAAAiwggKAuoIB4A== n’a pu être décodée. index.html
  La ressource multimédia data:video/webm;base64,GkXfowEAAAAAAAAfQoaBAUL3gQFC8oEEQvOBCEKChHdlYm1Ch4ECQoWBAhhTgGcB/////////xFNm3QBAAAAAAAAABVJqWYBAAAAAAAASSrXsYMPQkBEiYgAAAAAAAAAAE2AmFFUbXV4aW5nQXBwTGliV2ViTS0wLjAuMVdBmVFUd3JpdGluZ0FwcExpYldlYk0tMC4wLjEWVK5rAQAAAAAAADWuAQAAAAAAACzXgQFzxYQ+kgeYJYaIg1ZQOIOBAYaFVl9WUDjgAQAAAAAAAAiwggKAuoIB4A== n’a pas pu être décodée, erreur : Error Code: NS_ERROR_DOM_MEDIA_METADATA_ERR (0x806e0006)

```

Plus, could you remove `package.json` and `package-lock.json` from your PR.

**Comment by ashish0910** [comment]
@llaske now it shall work on all platforms.

**Comment by llaske** [comment]
Unfortunately I've got the same issue on Firefox.

**Comment by ashish0910** [comment]
Works perfectly on my Firefox without any error on console . can you please try again 2-3 times as the other you were getting will be solved by just restarting your browser . ref - [ https://stackoverflow.com/questions/44780856/firefox-html5-audio-media-resource-could-not-be-decoded-onmediasinkaudioerror ](url) 

sugar

**Comment by llaske** [comment]
Hmmm. Seems to works now. Strange.
BTW there is another issue: the security popup is shown two times (like in your video). Could you fix that?

**Comment by ashish0910** [comment]
@llaske we are getting pop up twice because we are making call to the camera twice(one for the preview and other for capture) and since we are testing the project on localhost , browser does not remember that we have granted the access and asks again . However on HTTPS this issue would not occur and browser will remember our choice .

**Comment by ashish0910** [comment]
@llaske everything works perfectly now .

File: activities/Record.activity/js/activity.js [hunk 1]
```diff
@@ -66,10 +62,10 @@ define(["sugar-web/activity/activity","sugar-web/presence","activity/capture-hel
             if(vidDisplay.style.display == "none"){
                 vidDisplay.style.display = "block" ;
             }
-            navigator.getUserMedia = navigator.getUserMedia || navigator.webkitGetMedia || navigator.mozGetUserMedia || navigator.msGetUserMedia || navigator.oGetUserMedia ;
-            if(navigator.getUserMedia){
-                navigator.getUserMedia({video:true},handleVideo,videoError);
-            }
+            // navigator.getUserMedia = navigator.getUserMedia || navigator.webkitGetMedia || navigator.mozGetUserMedia || navigator.msGetUserMedia || navigator.oGetUserMedia ;
```
Comment (llaske) [inline activities/Record.activity/js/activity.js]: You must not leave commented code

**Comment by llaske** [comment]
Nice. It works now.
Please remove commented code.

**Comment by ashish0910** [comment]
done

**Comment by llaske** [comment]
Good work. Thanks.

**Pull request merged**